lib/init.coffee

```diff
@@ -27,7 +27,8 @@ module.exports =
       wd = path.dirname filePath
       # Use the text editor's working directory as the classpath.
       #  TODO: Make the classpath user configurable.
-      args = ['-Xlint:all', '-cp', wd, filePath]
+      classpath = process.env.CLASSPATH
+      args = ['-Xlint:all', '-cp', wd + ":" + classpath, filePath]
```

Comment:
Done, I thought that it was the same char for Linux and Windows but I was wrong ;)